audio-recorder.js

```diff
@@ -177,12 +165,9 @@ module.exports = class AudioRecorder {
       throw new Error('No live audio input available or permitted');
     }
 
-    if (this.canMediaDevicesGetUserMedia) {
-      // Use of promises is required.
-      navigator.mediaDevices.getUserMedia({audio: true}).then(success).catch(failure);
-    } else if (this.canGetUserMedia) {
-      navigator.getUserMedia({audio: true}, success, failure);
-    }
+    getUserMedia({audio: true})
+      .then(success)
```

Comment:
you call getUserMedia above too, you could combine the then and catch clauses. Why do it twice?